frontend/src/components/NavBar/NavBar.css

```diff
@@ -33,15 +33,26 @@
 }
 
 .right {
+  display: flex;
```

Comment:
You should convert px to rem so the sizes of your components scale with the user's font size.

Reply:
Remember to do "npm run format" before you push the code.

Reply:
Thanks for the feedback! What does rem do? I tried changing it from px to rem but I think I broke something lol

Reply:
I believe 'rem' is a unit relative to the root element's font size, so if the user scales the font, everything else would get scaled properly. This was something my TPM told me last semester. You can read more about it [here](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px). Remember to [convert](https://nekocalc.com/px-to-rem-converter) your px to rem, you cannot simply change the unit.

Reply:
Finished converting px -> rem